charts/cheetah-application/Chart.yaml

```
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.3
+version: 0.6.4
```

It probably makes sense to make this a minor change instead:
```suggestion
version: 0.7.0
```